### DISCUSSION TITLE
How to show alerts in custom dashboard?

### DISCUSSION THREAD

**Discussion by shalak** [netdata/netdata#14393] [Q&A] [answered]
Is there a way to show alerts, or at least alert count in custom dashboards? The [custom dashboard documentation](https://learn.netdata.cloud/docs/agent/web/gui/custom) doesn't mention anything and I cannot see anything alert-related in `http://your.netdata.server:19999/api/v1/charts` response.

**Comment by andrewm4894** [comment, marked as answer]
What i have been doing is using the [alarms collector](https://learn.netdata.cloud/docs/agent/collectors/python.d.plugin/alarms) to essentially pull from `api/v1/charts` and then plot alert status (and value if wanted) over time just like any other chart. Then you can easily add that to a custom dashboard like any other chart.

Then for example, on one of my ml demo nodes i can see that one particular alert keeps going into warning (0 -> 1) every now and then:

![image](https://user-images.githubusercontent.com/2178292/216042937-2ef69c2d-676f-492c-8292-0c491a63efe2.png)

We are actually also working on a new feature right now that brings something more like the alert log into a wider "feed" within netdata.

I believe plan is for it to be available for dogfooding soon and will look something like this: 
![image](https://user-images.githubusercontent.com/2178292/216042152-50c53df2-63e6-4e5c-a68d-ce5c20c4c823.png)

Not actually sure if will be able to be added to custom dashboards but think that could be a nice feature request. 

@hugovalente-pm please correct me if im wrong on any of this.

**Reply by shalak** [reply]
This is great, thanks! I guess now I have to design some kind of gauge/piechart showing "Amount of errors in the last N hours", so I can have nice badges for my homepage ;)
